ctags.go

```diff
@@ -269,7 +258,7 @@ func (p *ctagsProcess) Parse(name string, content []byte) ([]*Entry, error) {
 				Signature:  rep.Signature,
 			})
 		default:
-			return nil, fmt.Errorf("ctags unexpected response %s for %s", rep.Typ, name)
+			return nil, &ParseError{Message: fmt.Sprintf("ctags unexpected response %s", rep.Typ), Fatal: true}
```

Comment:
I removed all handling of file names in the error logic. The `Parse` method is called with a single file, so the caller can report the failing file if they want (it doesn't need to be the responsibility of go-ctags errors).